--- a/verif/tb/core/uvma_cva6pkg_utils.sv	2026-08-18 08:32:26.389370447 -0700
+++ b/verif/tb/core/uvma_cva6pkg_utils.sv	2026-08-18 08:32:26.394079420 -0700
@@ -78,7 +78,7 @@
 
     void'(spike_set_param_bool(base, "hide_csrs_based_on_priv", 1));
     void'(spike_set_param_uint64_t(base, "mtvec_vectored_alignment", 64 * 4));
-    void'(spike_set_param_str(base, "extensions", "cvxif"));
+    void'(spike_set_param_str(base, "extensions", "parser"));
 
     // All enabled except XS and TW bits
     void'(spike_set_param_uint64_t(base, "mstatus_write_mask", 'hFFDE_7FFF));
